server/clip_server/executors/clip_onnx.py

```diff
@@ -27,6 +27,20 @@ def __init__(
         model_path: Optional[str] = None,
         **kwargs,
     ):
+        """
+        :param name: The name of the model to be used. Default 'ViT-B-32::openai'. A list of available models can be
+            found at https://clip-as-service.jina.ai/user-guides/server/#model-support
+        :param device: 'cpu' or 'cuda'. Default is None, which auto-detects the device.
+        :param num_worker_preprocess: The number of CPU workers to preprocess images and texts. Default is 4.
+        :param minibatch_size: The size of the minibatch for preprocessing and encoding. Default is 32. Reduce this
+            number if you encounter OOM errors.
+        :param access_paths: Default access paths for encoding, used if the access path is not passed as a parameter
```

Comment:
The access paths to traverse on the input documents to get the image and text documents to be processed.

Reply:
done